code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -536,6 +536,14 @@
 	race = /datum/species/plasmaman
 	mutationtext = "<span class='danger'>The pain subsides. You feel... flammable.</span>"
 
+/datum/reagent/stableslimetoxin/moth
```

Comment:
This collides with @XDTM clean-out of race change chemicals in #34166 remove this and re-add if their rework PR fails or seek approval to add it i guess if it passes.

Reply:
There is no official authority on conflicts from unrelated PRs when I last saw a case of this, it is at the discretion of the PR authors to do as they please. They're not entitled to change their PR for the sake of a different PR that isn't even in or guaranteed to get in.

Reply:
If you read the linked commit i included @Xhuis  [it does not follow the changes XDTM makes for this section of the code](https://github.com/tgstation/tgstation/pull/34166/commits/d191f9f3a3cbb4e0dbf2136843be8b70608b948f#diff-37c9a4b0e01ce56486c150228c427e14) , It is clearly conflicting with @XDTM 's PR work hence why i ask for them to try and work it out on a medium like IRC so they don't clash because they _are_ related.

I would think a maintainer would step in to do this kind of thing in that case Xhuis but we've both begun to github-lawyer now so lets not stop just yet.

Reply:
I... know what you're pointing out. I'm just saying that neither side is forced to fix conflicts just for the other's PR unless they want to, as has been shown in the past. I haven't ever seen a maintainer take sides for it unless the conflicting PR is a very large one.

Reply:
dont' reply to fwoosh posts